src/core_modules/capture-core/converters/clientToList.js

```diff
@@ -92,7 +92,7 @@ const valueConvertersForType = {
     [dataElementTypes.FILE_RESOURCE]: convertResourceForDisplay,
     [dataElementTypes.IMAGE]: convertResourceForDisplay,
     [dataElementTypes.ORGANISATION_UNIT]: (rawValue: Object) => rawValue.name,
-    [dataElementTypes.ASSIGNEE]: (rawValue: Object) => `${rawValue.name} (${rawValue.username})`,
+    [dataElementTypes.ASSIGNEE]: (rawValue: Object) => `${rawValue.name || rawValue.displayName} (${rawValue.username})`,
```

Comment:
**NIT**: There is an edge case where `displayName` is undefined. To reproduce, create a new enrollment event, you will be redirected to the enrollment overview and the `undefined (undefined)` text will appear in the program stage table. You must refresh the page to see the correct value of `displayName`. 

<img width="893" alt="Screenshot 2023-11-01 at 15 58 47" src="https://github.com/dhis2/capture-app/assets/6817073/bd6b7f38-6f1d-413e-afd5-f472317ebd72">


This happens because only the `uid` property is stored in Redux (within `enrollmentDomain`) when the event is created. The solution is quite straightforward and I already needed to fix it in one of the other related tickets in the epic [DHIS2-15480](https://github.com/dhis2/capture-app/pull/3412/files#diff-af963d544e60c1aea70db6cc2be6e5cd4f0bb4901cdbf40de87648e3b5a2c45bR29).

Thanks!

Reply:
Thanks @simonadomnisoru! Implemented the fix so that it won't come up as an issue during testing. I tried undoing the change I did on this line, but that didn't work. Seems like removing `rawValue.name` works, though. Do you think I should?

Reply:
To undo the change, you can add a converter for `dataElementTypes.ASSIGNEE` inside [serverToClient](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/converters/serverToClient.js#L38). Something like this should work:

```
    [dataElementTypes.ASSIGNEE]: (serverValue: Object) => ({
        id: serverValue.uid,
        name: serverValue.displayName,
        username: serverValue.username,
        firstName: serverValue.firstName,
        surname: serverValue.surname,
    }),
```
This ensures that when the assignedUser value reaches the `clientToList` converter, it is in the correct format. Thanks!

Reply:
No luck with that one.. I suppose we'd better keep both `rawValue.name